stat.go

```diff
@@ -648,6 +648,94 @@ func KullbackLeibler(p, q []float64) float64 {
 	return kl
 }
 
+// LinearRegression computes the best-fit line
+//  y = alpha*x + beta
```

Comment:
This documentation is the opposite of the code. The code computes alpha as the offset and beta as the slope. Personally, I think it's nicer to return the slope first.